### PR TITLE
python3Packages.hwi: init at 1.1.2

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

New package to interact with Hardware Wallets

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/hwi/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+buildPythonPackage rec {
+  pname = "hwi";
+  version = "1.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "eec460a51eb556500c1eca92015be246d5714cd53171407a76da71e4346048ae";
+  };
+
+  propagatedBuildInputs = [ mnemonic ecdsa typing-extensions hidapi libusb1 pyaes trezor btchip ckcc-protocol ];
```
Comment (jonringer) [inline pkgs/development/python-modules/hwi/default.nix:24]: with so many inputs, do you mind listing one per line
```suggestion
  propagatedBuildInputs = [
    mnemonic
    ecdsa
    ...
  ];
```

**Review by jonringer — COMMENTED** [review]
noticed that hwi is exposed, is this normally used a python application? if so, please expose the command in `all-packages.nix`

```
  hwi = with python3Packages; toPythonApplication hwi;
```

File: pkgs/development/python-modules/hwi/default.nix (hunk 1 above)
Comment (prusnak) [reply pkgs/development/python-modules/hwi/default.nix]: Fixed in 12d262130bd4153ad77930b243f9257f37c2795f

**Comment by prusnak** [comment]
>  if so, please expose the command in ...

Done in 12d262130bd4153ad77930b243f9257f37c2795f

**Review by jonringer — APPROVED** [review]
LGTM
shows usage

```
[4 built, 44 copied (14.8 MiB), 2.7 MiB DL]
https://github.com/NixOS/nixpkgs/pull/93182
2 packages built:
hwi python37Packages.hwi
```

**Pull request merged**